src/patternfly/components/Page/page.scss

```
@@ -174,6 +176,12 @@ $pf-page-v6--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg
   --#{$page}--m-dock__main-container--MaxHeight: calc(100% - var(--pf-t--global--spacer--lg) * 2);
   --#{$page}--m-dock__main-container--MarginBlockStart: var(--pf-t--global--spacer--lg);
   --#{$page}--m-dock--ColumnGap: var(--pf-t--global--spacer--inset--page-chrome);
+
+  :where(.pf-v6-theme-glass) & {
```

Stylelint (17.4.0)

[error] Unexpected missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)